Doc update for private_ip_mode

themantissa opened the pull request:
mongodbatlas_private_ip_mode is now a Deprecated Feature.  Updated peering and private_ip_mode resource pages.

themantissa committed:
Doc update for private_ip_mode

mongodbatlas_private_ip_mode is now a Deprecated Feature.  Updated peering and private_ip_mode resource pages.

PacoDw commented on website/docs/r/network_peering.html.markdown:
```diff
@@ -111,11 +104,6 @@ resource "mongodbatlas_network_container" "test" {
   region           = "US_WEST"
 }
```
It's missing one GCP resource to set the correct TF configuration. I placed examples with GCP and Azure on the PR #181, Maybe it would be great to update those examples, we can update them on the #181 PR or here what do you think?

themantissa replied:
Ah, good point.   I could update the examples here from what you put in PR 181 (which helped me remember to do this update).    I'll do that now!

PacoDw replied:
Nice :) glad to help

themantissa replied:
Added with related changes and tried to capture what you reported about GCP.  Please check it out @PacoDw.

PacoDw replied:
Thank you :)

PacoDw replied:
Travis marks two spelling errors in the following lines:
```bash
website/docs/r/network_peering.html.markdown:40:8: "RESOURSE" is a misspelling of "RESOURCES"
230website/docs/r/network_peering.html.markdown:149:41: "RESOURSE" is a misspelling of "RESOURCES"
```

themantissa replied:
Thank you, fixed them and passed 💯

themantissa committed:
Add new examples for GCP/Azure

PacoDw reviewed (approved):
Pretty cool! Thank you so much :)

themantissa committed:
spelling fix

shum commented on website/docs/r/network_peering.html.markdown:
```diff
@@ -69,65 +75,99 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
+
+  replication_factor           = 3
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
```
should we use 4.2?

themantissa replied:
Yeah, should start moving all the examples to 4.2.  I'll start w/ here and update the rest soon.

shum commented on website/docs/r/network_peering.html.markdown:
```diff
@@ -69,65 +75,99 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
+
+  replication_factor           = 3
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
```
ditto, 4.2?

shum commented on website/docs/r/network_peering.html.markdown:
```diff
@@ -69,65 +75,99 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
-  depends_on = [mongodbatlas_private_ip_mode.my_private_ip_mode]
+  //Provider Settings "block"
+  provider_name               = "AZURE"
+  provider_disk_type_name     = "P6"
```
could probably start with a P4 for 32GB storage only

themantissa replied:
makes sense, that's the default.  Will change

shum reviewed (approved):
Small parameter suggestions.

themantissa committed:
Edit defaults in examples

Edit to start moving default version to 4.2 and Azure disk to P4, also the default.
